tests/kafkatest/services/verifiable_producer.py

```diff
@@ -37,7 +39,19 @@ def __init__(self, context, num_nodes, kafka, topic, max_messages=-1, throughput
         self.acked_values = []
         self.not_acked_values = []
 
+        # If security protocol is not specified, use the default in the template properties.
+        self.prop_file = self.render('verifiable_producer.properties')
+        self.security_config = SecurityConfig(security_protocol, self.prop_file)
```

Comment:
@granders @guozhang The advantage of associating property files with all services like the verifiable producer is that it makes it possible to run producers with different default properties without modifying python code. We are planning to use this flexibility to run system tests with the security configuration we use (eg. endpoint validation on, client auth off, specific TLS version etc). If empty property files are an issue, would it be possible to change TemplateRenderer.render() in ducktape to treat missing property file as an empty file rather than remove support for property files in some services?